src/log-worker/consumeMessage.ts

```diff
@@ -0,0 +1,15 @@
+import Worker = require('tortoise');
+import { DotenvParseOutput } from 'dotenv';
+
+export const consumeMessage = async (
+  config: DotenvParseOutput,
+  worker: Worker,
+  callback: (message: string, acknowledgment: () => Promise<void>) => void
+): Promise<void> => {
+  const options: object = { durable: config.QUEUE_DURABLE || true };
+
+  await worker
+    .queue(config.QUEUE_NAME || 'ADX_LOG_WORKER', options)
```

Comment:
config.QUEUE_NAME || 'ADX_LOG_WORKER' could go in a const named something like queueName. Function calls should not have logic embedded, what do you say 😅.

The same applies to config.QUEUE_DURABLE || true

Reply:
ait.

Reply:
done